File: sample-chat-java/app/src/main/java/com/quickblox/sample/chat/java/utils/imagepick/ImagePickHelper.java
package com.quickblox.sample.chat.java.utils.imagepick;

import com.quickblox.sample.chat.java.utils.imagepick.fragment.MediaPickHelperFragment;
import com.quickblox.sample.chat.java.utils.imagepick.fragment.MediaSourcePickDialogFragment;

import androidx.fragment.app.FragmentActivity;
import androidx.fragment.app.FragmentManager;

public class ImagePickHelper {

    public void pickAnImage(FragmentActivity activity, int requestCode) {
        MediaPickHelperFragment mediaPickHelperFragment = MediaPickHelperFragment.start(activity, requestCode);
        showImageSourcePickerDialog(activity.getSupportFragmentManager(), mediaPickHelperFragment);
    }

    private void showImageSourcePickerDialog(FragmentManager fm, MediaPickHelperFragment fragment) {
        MediaSourcePickDialogFragment.show(fm,
                new MediaSourcePickDialogFragment.ImageSourcePickedListener(fragment));
    }
}

File: sample-chat-kotlin/app/src/main/java/com/quickblox/sample/chat/kotlin/utils/imagepick/ImagePickHelper.kt
package com.quickblox.sample.chat.kotlin.utils.imagepick

import androidx.fragment.app.FragmentActivity
import androidx.fragment.app.FragmentManager
import com.quickblox.sample.chat.kotlin.utils.imagepick.fragment.MediaPickHelperFragment
import com.quickblox.sample.chat.kotlin.utils.imagepick.fragment.MediaSourcePickDialogFragment


fun pickAnImage(activity: FragmentActivity, requestCode: Int) {
    val mediaPickHelperFragment = MediaPickHelperFragment.getInstance(activity, requestCode)
    showImageSourcePickerDialog(activity.supportFragmentManager, mediaPickHelperFragment)
}

private fun showImageSourcePickerDialog(fragmentManager: FragmentManager, fragment: MediaPickHelperFragment) {
    MediaSourcePickDialogFragment.show(fragmentManager,
            MediaSourcePickDialogFragment.ImageSourcePickedListener(fragment))
}